### PR TITLE
Add free properties in HVDC forms.

### PR DESCRIPTION
**Pull request by flomillot**
Also extract anf factorize getConcatenatedProperties.

### CODE REVIEW

**Review by Mathieu-Deharbe — CHANGES_REQUESTED** [review]
When an HVDC is loaded in the HVDC modification interface, the contained properties are not loaded (or, more likely, loaded but not displayed).

When you modify a battery which has some properties, they are loaded and displayed in the additional property part : 

modif batterie

This is not the case for this HVDV (that I created in the previous node, giving it a property that can be observed in the spreadsheet) : 

modif HVDC

---------------------------
Second strange problem but probably not because of your dev : if I create a HVDC in a node, I can load it in the HVDC modification interface. But I can't find and load it in the HVDC creation interface. It is simply not in the list.

**Comment by flomillot** [comment]
> When an HVDC is loaded in the HVDC modification interface, the contained properties are not loaded (or, more likely, loaded but not displayed).
> 
> When you modify a battery which has some properties, they are loaded and displayed in the additional property part :
> 
> modif batterie
> 
> This is not the case for this HVDC (that I created in the previous node, giving it a property that can be observed in the spreadsheet) :
> 
> modif HVDC
> 
> Second strange problem but probably not because of your dev : if I create a HVDC in a node, I can load it in the HVDC **modification** interface. But I can't find and load it in the HVDC **creation** interface. It is simply not in the list.

Actually it was not loaded because I had to change network-map-server also : https://github.com/gridsuite/network-map-server/pull/187

File: src/components/dialogs/network-modifications/common/properties/property-utils.ts [hunk 1]
```diff
@@ -120,6 +120,18 @@ export const copyEquipmentPropertiesForCreation = (
     };
 };
 
+export function getConcatenatedProperties(
+    equipment: any,
```
[the remark this answers is not in the document]
Comment (flomillot) [reply src/components/dialogs/network-modifications/common/properties/property-utils.ts]: Yes you're right, thanks

**Review by Mathieu-Deharbe — APPROVED** [review]
OK for me.

I am going to report the small loading bug on slack.

File: src/components/dialogs/network-modifications/vsc/hvdc-line-pane/vsc-hvdc-line-pane.tsx [hunk 2]
```diff
@@ -220,6 +221,8 @@ const VscHvdcLinePane: FunctionComponent<VscHvdcLinePaneProps> = ({
                 {gridItem(droopField, 6)}
                 {gridItem(p0Field, 6)}
             </Grid>
+
+            <PropertiesForm isModification={isEquipementModification} />
```
Comment (ghazwarhili) [inline src/components/dialogs/network-modifications/vsc/hvdc-line-pane/vsc-hvdc-line-pane.tsx:225]: networkElementType ?

Comment (flomillot) [reply src/components/dialogs/network-modifications/vsc/hvdc-line-pane/vsc-hvdc-line-pane.tsx:225]: Not for HVDC, that's why I changed the code of PropertiesForm

**Review by ghazwarhili — APPROVED** [review]

**Pull request merged**